Project1/style.css

```
@@ -1,13 +1,44 @@
 body {
-    background-color: #9cbabc;
+    background-color: #d2d3d3;
+
+}
+
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    color: #b5afc3
```

Remember to end every declaration with a semicolon:
```suggestion
    color: #b5afc3;
```